leaf-ui/plugins/slider/nouislider.min.css

```diff
@@ -1,4 +1,316 @@
+    -webkit-transition:background 450ms;
+    transition:background 450ms
+
+}
+
+.noUi-origin{
+    border-radius:2px
+
+}
+
+.noUi-target{
+    border-radius:4px;
+    border:1px solid #D3D3D3;
+    box-shadow:inset 0 1px 1px #F0F0F0,0 3px 6px -5px #BBB
+
+}
+
+.noUi-target.noUi-connect{
+    box-shadow:inset 0 0 3px rgba(51,51,51,.45),0 3px 6px -5px #BBB
+
+}
+
+.noUi-draggable{
+    cursor:w-resize
+
+}
+
+.noUi-vertical .noUi-draggable{
+    cursor:n-resize
+
+}
+
+.noUi-handle{
+    border:1px solid #D9D9D9;
+    border-radius:3px;
+    background:#FFF;
+    cursor:default;
+    box-shadow:inset 0 0 1px #FFF,inset 0 1px 7px #EBEBEB,0 3px 6px -3px #BBB
+
+}
+
+.noUi-active{
+    box-shadow:inset 0 0 1px #FFF,inset 0 1px 7px #DDD,0 3px 6px -3px #BBB
+
+}
+
+.noUi-handle:after,.noUi-handle:before{
+    content:"";
+    display:block;
+    position:absolute;
+    height:14px;
+    width:1px;
+    background:#E8E7E6;
+    left:14px;
+    top:6px
+
+}
+
+.noUi-handle:after{
+    left:17px
+
+}
+
+.noUi-vertical .noUi-handle:after,.noUi-vertical .noUi-handle:before{
+    width:14px;
+    height:1px;
+    left:6px;
+    top:14px
+
+}
+.noUi-vertical .noUi-handle:after{
+    top:17px
+
+}
+
+[disabled] .noUi-connect,[disabled].noUi-connect{
+    background:#B8B8B8
+
+}
+
+[disabled] .noUi-handle,[disabled].noUi-origin{
+    cursor:not-allowed
+
+}
+
+.noUi-pips,.noUi-pips *{
+    -moz-box-sizing:border-box;
+    box-sizing:border-box
+
+}
+
+.noUi-pips{
+    position:absolute;
+    color:#999
+
+}
+
+.noUi-value{
+    width:40px;
+    position:absolute;
+    text-align:center
+}
+
+.noUi-value-sub{
+    color:#ccc;
+    font-size:10px
+
+}
+
+.noUi-marker{
+    position:absolute;
+    background:#CCC
+
+}
+
+.noUi-marker-large,.noUi-marker-sub{
+    background:#AAA
+
+}
+.noUi-pips-horizontal{
+    padding:10px 0;
+    height:50px;
+    top:100%;
+    left:0;
+    width:100%
+
+}
+
+.noUi-value-horizontal{
+    margin-left:-20px;
+    padding-top:20px
+
+}
+
+.noUi-value-horizontal.noUi-value-sub{
+    padding-top:15px
+
+}
+
+.noUi-marker-horizontal.noUi-marker{
+    margin-left:-1px;
+    width:2px;
+    height:5px
+
+}
+
+.noUi-marker-horizontal.noUi-marker-sub{
+    height:10px
+
+}
+
+.noUi-marker-horizontal.noUi-marker-large{
+    height:15px
+
+}
+
+.noUi-pips-vertical{
+    padding:0 10px;
+    height:100%;
+    top:0;
+    left:100%
+
+}
+
+.noUi-value-vertical{
+    width:15px;
+    margin-left:20px;
+    margin-top:-5px
+
+}
+
+.noUi-marker-vertical.noUi-marker{
+    width:5px;
+    height:2px;
+    margin-top:-1px
+
+}
+
+.noUi-marker-vertical.noUi-marker-sub{
+    width:10px
+
+}
+
+.noUi-marker-vertical.noUi-marker-large{
+    width:15px
+
+}
+
+.noUi-tooltip{
+    display:block;
+    position:absolute;
+    border:1px solid #D9D9D9;
+    border-radius:3px;
+    background:#fff;
+    padding:5px;
+    text-align:center
+
+}
+
+.noUi-horizontal .noUi-handle-lower .noUi-tooltip{
+    top:-32px

```

Comment:
Do you need` ; `at the end? 

If so, there are some lines here and there that don't have the semicolon while others do.